Module_Postman/activities/Act5_Mock_Servers/1.md

```diff
@@ -0,0 +1,17 @@
+<!--title={Introduction to Mock Servers}-->
+
+A **mock server** is a fake API that can simulate server responses without having to completely set up endpoints in the **backend** beforehand. 
+
+Now, why would someone use a mock server? Mock servers allow developers to see the what kind of responses they receive in order to further develop their new endpoints to display the desired behavior, and Postman offers the option to create either public or private servers, with which users can control who has permission to access and edit the collection.
+
+Mock servers are often used towards the beginning of the API development process, as can be seen in the API Lifestyle diagram posted on the Postman website.
+
+![workflow](workflow.png)
```

Comment:
image location needing fixed here? or do these maybe only render in the live site..?